<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HATI-SDG00096" failures="0" tests="3" name="OPDTest.Cancel_new_appointment_against_Doctor" time="193.985" errors="0" timestamp="2024-07-29T11:43:50 IST" skipped="0">
  <testcase classname="OPDTest.Cancel_new_appointment_against_Doctor" name="TC_01_Verify_that_user_is_able_to_login_the_application" time="14.569"/>
  <testcase classname="OPDTest.Cancel_new_appointment_against_Doctor" name="TC_02_Verify_that_user_is_able_to_book_the_New_appointment" time="124.685"/>
  <testcase classname="OPDTest.Cancel_new_appointment_against_Doctor" name="TC_03_Verify_that_appointment_is_cancelled_succsessfull" time="54.731"/>
</testsuite> <!-- OPDTest.Cancel_new_appointment_against_Doctor -->
